addons/mail/static/src/components/message_list/message_list.scss

```diff
@@ -1,121 +1,13 @@
-// ------------------------------------------------------------------
-// Layout
-// ------------------------------------------------------------------
-
-.o_MessageList {
-    display: flex;
-    flex-flow: column;
-    overflow: auto;
-
-    &.o-empty {
-        align-items: center;
-        justify-content: center;
-    }
-
-    &:not(.o-empty) {
-        padding-bottom: map-get($spacers, 4);
-    }
-}
-
-.o_MessageList_empty {
-    flex: 1 1 auto;
-    height: map-get($sizes, 100);
-    width: map-get($sizes, 100);
-    align-self: center;
-    display: flex;
-    flex-flow: column;
-    align-items: center;
-    justify-content: center;
-    padding: map-get($spacers, 4);
-    line-height: 2.5rem;
```

Comment:
To keep the same result, I replaced this `line-height` by a margin class on `o_MessageList_emptyTitle`.